spec/domain/healthchecks/monthly_aggregations_spec.rb

```diff
@@ -0,0 +1,26 @@
+RSpec.describe Healthchecks::MonthlyAggregations do
+  include_examples 'Healthcheck enabled/disabled within time range'
+
+  around do |example|
+    Timecop.freeze(Date.new(2019, 2, 22)) { example.run }
+  end
+
+  its(:name) { is_expected.to eq(:aggregations) }
+
+  describe '#status' do
+    context 'When there are aggregations' do
```

Comment:
Could you please make it explicit that they are `monthly` aggregations?

Reply:
Thanks - addressed in https://github.com/alphagov/content-performance-manager/pull/1153/commits/01a92df30f727fd33857bba12d4ada76cd6471ae